peer/example_test.go

```diff
@@ -38,11 +45,16 @@ func mockRemotePeer() error {
 			return
 		}
 
-		// Create and start the inbound peer.
+		// Create and start the inbound
 		p := peer.NewInboundPeer(peerCfg)
-		p.AssociateConnection(conn)
+		err = p.AssociateConnection(conn)
+		if err != nil {
+			fmt.Printf("AssociateConnection: error %+v\n", err)
+			return
+		}
 	}()
 
+	<-time.After(1 * time.Second)
```

Comment:
I'm not sure I'm a fan of this. Every time a test calls this function the whole test runtime becomes 1 second slower.

Reply:
Oh yeah, it's a terrible idea in all 100%. Added it when checking something and forgot to remove.

Good catch!